src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocMethodCheck.java

```diff
@@ -721,16 +721,20 @@ private static int calculateTagColumn(Matcher javadocTagMatcher,
         final DetailAST throwsAST = ast
                 .findFirstToken(TokenTypes.LITERAL_THROWS);
         if (throwsAST != null) {
-            DetailAST child = throwsAST.getFirstChild();
-            while (child != null) {
-                if (child.getType() == TokenTypes.IDENT
-                        || child.getType() == TokenTypes.DOT) {
+            DetailAST exception = throwsAST.getFirstChild();
+            while (exception != null) {
+                if (exception.getType() == TokenTypes.IDENT
+                        || exception.getType() == TokenTypes.DOT) {
+                    DetailAST child = exception;
                     final FullIdent ident = FullIdent.createFullIdent(child);
-                    final ExceptionInfo exceptionInfo = new ExceptionInfo(
+                    while (child.getType() == TokenTypes.DOT) {
+                        child = child.getFirstChild();
+                    }
```

Comment:
Please explain why do you need this logic?

Reply:
this is because there is a special case:
https://github.com/HuGanghui/checkstyle/blob/f88da2028c8167f4b7b7ef7f9fd6350a5f69b5ec/src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocmethod/InputJavadocMethodExtraThrows.java#L77-L82